current-sdk-versions.md

```diff
@@ -0,0 +1,142 @@
+# Current SDK Versions
+
+{% hint style="warning" %}
+## Migrate to Swift
+
+If you are currently using our Objective-C \(ACP-prefix libraries\), please see [the guide for migrating to Swift](../migrate-to-swift.md) for next steps. The AEP Swift iOS SDKs are open source - read more about [our move to Swift and open source](https://medium.com/adobetech/adobe-experience-platform-mobile-sdks-move-to-swift-for-ios-6aa67b67b4d4).
+{% endhint %}
+
+## iOS (ACP libraries)
+
+{% hint style="info" %}
+The iOS Adobe Experience Platform Mobile SDKs support iOS 10 or later \(includes support for iOS, iPadOS, and tvOS\).
+{% endhint %}
+
+| Extension | Cocoapods | Github |
+| :--- | :--- | :--- |
+| [Mobile Core](./foundation-extensions/mobile-core) \(supports tvOS\) | [![Cocoapods](https://img.shields.io/cocoapods/v/ACPCore.svg?color=orange&label=ACPCore&logo=apple&logoColor=white&style=flat-square)](https://cocoapods.org/pods/ACPCore) | [Link](https://github.com/Adobe-Marketing-Cloud/acp-sdks/tree/master/iOS/ACPCore) |
+| [Profile](./foundation-extensions/profile) | [![Cocoapods](https://img.shields.io/cocoapods/v/ACPUserProfile.svg?color=orange&label=ACPUserProfile&logo=apple&logoColor=white&style=flat-square)](https://cocoapods.org/pods/ACPUserProfile) | [Link](https://github.com/Adobe-Marketing-Cloud/acp-sdks/tree/master/iOS/ACPUserProfile) |
+| [Identity](./foundation-extensions/mobile-core/identity/) | Bundled in Mobile Core | — |
+| [Signal](./foundation-extensions/mobile-core/signals/) | Bundled in Mobile Core | — |
+| [Lifecycle](./foundation-extensions/mobile-core/lifecycle/) | Bundled in Mobile Core | — |
+| [Rules Engine](./foundation-extensions/mobile-core/rules-engine) | Bundled in Mobile Core | — |
+| [Adobe Experience Platform Edge Network](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension) | Not Available | — |
+| [Identity for Edge Network](https://aep-sdks.gitbook.io/docs/foundation-extensions/identity-for-edge-network/) | Not Available | — |
+| [Consent for Edge Network](https://aep-sdks.gitbook.io/docs/foundation-extensions/consent-for-edge-network/) | Not Available | — |
```

Comment:
Wondering if we should just get rid of Edge, Identity for Edge and Consent here because with a separate branch for 2.x these wouldn't show up on left nav anyways. And I'm assuming with more new Swift extensions in the works, we do not want to come and update this table (e.g. Optimize)